<ui version="4.0" >
 <class>DialogAnalog</class>
 <widget class="QDialog" name="DialogAnalog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>368</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>4</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>4</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>4</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_dataview" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>7</hsizetype>
             <vsizetype>7</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="hlScrollBar" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>1</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="minimumSize" >
            <size>
             <width>70</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>160</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title" >
            <string>Source</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>2</number>
            </property>
            <property name="spacing" >
             <number>2</number>
            </property>
            <item>
             <widget class="QTreeWidget" name="twWatchPoint" >
              <property name="minimumSize" >
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbLoad" >
           <property name="text" >
            <string>Load Sample ...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSave" >
           <property name="text" >
            <string>Save Sample ...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>4</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>4</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chkBCapture" >
           <property name="text" >
            <string>CAPTURE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbMarker" >
           <property name="text" >
            <string>Insert Marker</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="chkBShowMarker" >
           <property name="text" >
            <string>Draw Markers</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbZoomIn" >
           <property name="text" >
            <string>Zoom +</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbZoomOut" >
           <property name="text" >
            <string>Zoom -</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbFit" >
           <property name="text" >
            <string>Fit Markers</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>4</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Markers length</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelLength" >
           <property name="text" >
            <string>ms</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <resources/>
 <connections>
  <connection>
   <sender>chkBShowMarker</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DialogAnalog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel" >
     <x>177</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
